### ISSUE TITLE
Importing with the default .env file failed

### ISSUE THREAD

**Issue by gereons** [solectrus/csv-importer#62]
I'm just setting up my new/first solectrus installation (and am VERY happy so far!) and came across this issue when importing my Senec CSV files:

```
CSV importer for SOLECTRUS, Version 0.3.1, built at 2023-07-02T05:57:19.952Z
https://github.com/solectrus/csv-importer
Copyright (c) 2020-2023 Georg Ledermann, released under the MIT License

bundler: failed to load command: app/main.rb (app/main.rb)
/csv-importer/app/config.rb:39:in `fetch': key not found: "INFLUX_MEASUREMENT" (KeyError)
Did you mean?  "INFLUX_MEASUREMENT_PV"
	from /csv-importer/app/config.rb:39:in `from_env'
	from app/main.rb:17:in `<top (required)>'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/cli/exec.rb:58:in `load'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/cli/exec.rb:58:in `kernel_load'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/cli/exec.rb:23:in `run'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/cli.rb:492:in `exec'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/vendor/thor/lib/thor.rb:392:in `dispatch'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/cli.rb:34:in `dispatch'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/vendor/thor/lib/thor/base.rb:485:in `start'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/cli.rb:28:in `start'
	from /usr/local/bundle/gems/bundler-2.4.15/exe/bundle:37:in `block in <top (required)>'
	from /usr/local/bundle/gems/bundler-2.4.15/lib/bundler/friendly_errors.rb:117:in `with_friendly_errors'
	from /usr/local/bundle/gems/bundler-2.4.15/exe/bundle:29:in `<top (required)>'
	from /usr/local/bundle/bin/bundle:25:in `load'
	from /usr/local/bundle/bin/bundle:25:in `<main>'
```

Indeed, my .env file (as per the default installation) only has a `INFLUX_MEASUREMENT_PV=SENEC` line. 

Duplicating this to add `INFLUX_MEASUREMENT=SENEC` and re-running the imported fixed the error for me, but this should probably be changed in the code as well.

**Comment by gereons** [comment]
PS: Just noticed that this was already addressed in #61, but this change does not seem to be released yet(?)

**Comment by ledermann** [comment]
Yes, a new release is overdue. I'm currently catching up on tasks that were left behind during my vacation :)

I just bumped the v0.3.2
